frontend/src/utils/colors.tsx

```diff
@@ -0,0 +1,8 @@
+import { colors } from '@material-ui/core';
+
+export const Colors = {
+    DEEP_RED: colors.red[900],
+    DEEP_BLUE: colors.blue[900],
+};
+
+export type ColorsType = typeof Colors['DEEP_RED'] | typeof Colors['DEEP_BLUE'];

```

Comment:
From now on, let's define app colors here.

Reply:
I think we would be better off utilizing material ui themes for this stuff. It w

Reply:
I don't have a strong opinion one way or the other

Reply:
This also doesn't really provide any abstraction of colors. For example, if I were to style a bunch of components using `DEEP_RED` and later decide I want them to be a pink, I now need to change the color on all the components to a new color. Alternatively, I could update the `DEEP_RED` color to pink but then the variable name would no longer reflect the actual color and may break other components using `DEEP_RED`. MUI theme avoids this problem by choosing names for the theme variables that reflect their use: https://material-ui.com/customization/theming/#api . This also has the consequence of making it really easy to change the color scheme of the entire application which I believe was a feature mentioned very early on during planning. I don't remember if we really cared about this feature that much though.

Reply:
I'll create a task to standardize coloring, since it's somewhat lower priority than the rest of the PR. But you make some good points, and I agree with you

Reply:
I'd argue that getting the theme stuff setup early should be fairly high priority. It will be annoying making the change later when a lot of the components have local styles that may or may not be cohesive with the rest of the site.

Perhaps we get a separate pull request going for setting up material UI theme context